core/src/main/java/org/apache/calcite/plan/Strong.java

```diff
@@ -356,7 +358,8 @@ private static Map<SqlKind, Policy> createPolicyMap() {
     map.put(SqlKind.CHECKED_TIMES, Policy.ANY);
     map.put(SqlKind.CHECKED_DIVIDE, Policy.ANY);
 
-    map.put(SqlKind.DIVIDE, Policy.ANY);
+    // DIVIDE behaves like ANY. However, (x/0) IS (NOT) NULL may not be simplified.
```

Comment:
This comment should say that in some dialects x/y may be NULL even when neither of x and y are NULL, e.g., y = 0.
it has nothing to do with simplification.